packages/vital-content-listing/src/components/ContentListing/tokens/vitalContentListing.ts

```diff
@@ -12,19 +12,21 @@
  * limitations under the License.
  */
 
-import { withNodeKey } from '@bodiless/core';
 import { vitalContentListingFlowContainer } from '../../ContentListingFlowContainer';
 import { vitalFilterByGroup } from '../../FilterByGroup';
 import { asContentListingToken } from '../ContentListingClean';
-import { ContentListingNodeKeys } from './constants';
 
 const Default = asContentListingToken({
   Components: {
     Wrapper: vitalFilterByGroup.Default,
     Content: vitalContentListingFlowContainer.Default,
   },
-  Schema: {
```

Comment:
Why are we removing this?

Reply:
@wodenx ,

Some issues happen when having more than one filter in the same page. I think we agree this is an edge case, probably that won't happen, however we are creating a page, other than the /styleguide, to demonstrate the filters (default, multiple, and sitewide) and there we are adding more than one filter.